ocrd_validators/ocrd_validators/workspace_validator.py

```diff
@@ -290,6 +298,7 @@ def _validate_page_xsd(self):
         """
         Validate all PAGE-XML files against PAGE XSD schema
         """
+        self.log.debug('_validate_page_xsd')
         log = getLogger('ocrd.workspace_validator')
         log.debug("Validating all PAGE-XML files against XSD")
```

Comment:
```suggestion
```

Reply:
plus further below (l. 299 / 308) there's a `log.debug` which should become `self.log.debug` or removed as well.